source/mainnet/net/references/multi-sig.rst

```diff
@@ -0,0 +1,448 @@
+
+To view the key directory path that the ``concordium-client`` tool is using, run the following command:
+
+   .. code-block:: console
+
+      $concordium-client config show
+
+The goal is to add additional keys for your account to the local key directory of the ``concordium-client``
+tool in this section (Option 2). The guide for the first option is in the
+:ref:`following section<sign-with-keyfile>`.
+
+.. dropdown:: Import the keys by adapting the key export file format from the browser wallet
+
+   We recommend that you export a key file from a random account on the browser wallet and use this
+   JSON file as a template to generate a corresponding JSON file for the keys of the account you want to update.
+
+   .. note::
+
+      You can read up on how to export a key file from the browser wallet :ref:`here<export-key>`.
+
+   Your ``browser_wallet.export`` file should look similar to the following browser wallet export template:
+
+   .. code-block:: json
+      :force:
+
+      {
+         "type": "concordium-browser-wallet-account",
+         "v": 0,
+         "environment": "testnet",
+         "value": {
+            "accountKeys": {
+               "keys": {
+                  "0": {
+                     "keys": {
+                        "0": {
+                           "signKey": "<Key_0_Private_Key_Without_0x_Prefix>",
+                           "verifyKey": "<Key_0_Public_Key_Without_0x_Prefix>"
+                        },
+                        "1": {
+                           "signKey": "<Key_1_Private_Key_Without_0x_Prefix>",
+                           "verifyKey": "<Key_1_Public_Key_Without_0x_Prefix>"
+                        }
+                     },
+                     "threshold": 2
+                  }
+               },
+               "threshold": 1
+            },
+            "credentials": {
+               "0": "97f325c9f86066ab0c80ff879c21629eb67818841940869308d6a72886d18f8668e62e43ad228fdcbda245d0722454df"
+            },
+            "address": "4jxvYasaPncfmCFCLZCvuL5cZuvR5HAQezCHZH7ZA7AGsRYpix"
+         }
+      }
+
+   You can import the keys of this ``browser_wallet.export`` file and associate it to your account as follows:
+
+   .. code-block:: console
+
+      $concordium-client config account import browser_wallet.export --name <choose-a-name-for-your-account>
+
+.. dropdown:: Assign an already imported key to your account
+
+   If you already imported the new key to ``concordium-client`` but it is associated with a different account
+   (e.g. because the public-private key pair was generated by creating a new account on Concordium which was imported
+   to ``concordium-client`` and this key pair was re-used in your multi-sig account),
+   you can look up the key in the local key directory of the ``concordium-client`` tool and associate
+   the file's content to your multi-sig account.
+
+   .. note::
+
+      We do not recommend to re-use keys on different accounts in production.
```

Comment:
What _do_ we recommend then? I feel like if we don't provide a recommended path for the user to follow, they will feel stuck.